### ISSUE TITLE
Falsepositive overridden_super_call

### ISSUE THREAD

**Issue by NikKovIos** [realm/SwiftLint#5655]
<img width="1292" alt="image" src="https://github.com/realm/SwiftLint/assets/16647690/d968eed5-e817-4fe9-af7e-579d6285ecd1">

```
    @IBOutlet private weak var contextView: UIView!
    override func viewWillAppear(_ animated: Bool) {
        if animated {
            contextView.transform = CGAffineTransform(translationX: 0, y: contextView.bounds.height)

            UIView.transition(with: contextView,
                              duration: 0.25,
                              options: .curveEaseInOut,
                              animations: {
                self.contextView.transform = .identity
            },
                              completion: { _ in
                super.viewWillAppear(true)
            })
        } else {
            super.viewWillAppear(false)
        }
    }
```

Version 0.55.1
